scripts/src/javatests/org/oppia/android/scripts/todo/CommentCheckTest.kt

```diff
@@ -0,0 +1,94 @@
+package org.oppia.android.scripts.todo
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TemporaryFolder
+import org.oppia.android.testing.assertThrows
+
+/** Tests for [CommentCheck]. */
+class CommentCheckTest {
+  private val permalinkPrefix = "https://github.com/oppia/oppia-android/blob/"
```

Comment:
Ditto here--suggest moving the last ``/`` to the later comments for slightly better readability (super nit).

Reply:
Done.